<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Penetrance Estimator</string>
  </property>
  <widget class="QWidget" name="MainWindowWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QWidget" name="VarsAndButtons" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="VarModelBox">
         <property name="title">
          <string>Model</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="VarModel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string notr="true">probAsct = const * r^k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>probAsct = const * (r^k + t) prob(aff|12)=beta</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>probAsct = const * (r^k + t) prob(aff|12)=alpha + beta - alpha * beta</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="VarAlphaBox">
         <property name="title">
          <string>alpha  ( prob(aff | '11') ) </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="VarAlpha">
            <property name="text">
             <string>0.001, 0.01, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.75</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="VarBetaBox">
         <property name="title">
          <string>beta  ( prob(aff | '12') )</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="VarBeta">
            <property name="text">
             <string>0.5</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="VarSBox">
         <property name="title">
          <string>s  (num of kids)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="VarS">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="VarKBox">
         <property name="title">
          <string>k</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="VarK">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>-1, 0, 1, 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="ButtonsFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="ButtonPlot">
            <property name="text">
             <string>Refresh Plot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Save plot image to file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonSave">
            <property name="text">
             <string>Save statistics to file</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="PenEstPlot" name="PlotWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>512</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(&quot;Figure_1.png&quot;);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PenEstPlot</class>
   <extends>QWidget</extends>
   <header>penestGUI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonPlot</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshPlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>savePlotImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveStatsFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshPlot()</slot>
  <slot>savePlotImage()</slot>
  <slot>saveStatsFile()</slot>
 </slots>
</ui>
